lib/razorpay/payment.rb

```diff
@@ -112,5 +112,9 @@ def self.create_upi(data={})
     def self.validate_vpa(data={})
       request.post "validate/vpa" , data
     end
+
+    def self.expend_details(id, options={})
```

Comment:
Can this function name be `expand_details` instead ?

Reply:
made correction in `payment.rb`, `payment.md` &  in unit test file

https://github.com/razorpay/razorpay-ruby/blob/75f75c0c47024f5d3f52238453c0d964bc3f23be/lib/razorpay/payment.rb#L116